Kernel/Jail.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/DistinctNumeric.h>
+#include <AK/Error.h>
+#include <AK/IntrusiveList.h>
+#include <AK/OwnPtr.h>
+#include <AK/RefPtr.h>
+#include <AK/Try.h>
+#include <AK/Types.h>
+#include <Kernel/KString.h>
+#include <Kernel/Library/LockRefPtr.h>
+#include <Kernel/Locking/SpinlockProtected.h>
+
+namespace Kernel {
+
+class JailManagement;
+
+AK_TYPEDEF_DISTINCT_ORDERED_ID(u64, JailIndex);
```

Comment:
If JailIndex is `u64`, then why does the syscall take `int`? that's not `u64` on any platform.